src/Tribe/Tracker.php

```diff
@@ -426,4 +429,16 @@ public function set_tracked_taxonomies( array $tracked_taxonomies ) {
 	public function set_tracked_post_types( array $tracked_post_types ) {
 		$this->tracked_post_types = $tracked_post_types;
 	}
+
+	/**
+	 * Make sure to remove the changed field if the event is deleted to ensure there are no left meta fields when
+	 * the event is deleted.
+	 *
+	 * @since TBD
+	 *
+	 * @param int Post ID
+	 */
+	public function cleanup_meta_fields( $post_id ) {
+		delete_post_meta( (int) $post_id, self::$field_key );
```

Comment:
Can we return the result of the `delete_post_meta()` and add the return to the Doc Block?

Reply:
Changes has been updated.